pontoon/base/urls.py

```diff
@@ -78,7 +78,7 @@
         views.get_translations_from_other_locales,
         name="pontoon.other_locales",
     ),
-    url(r"^download/", views.download, name="pontoon.download"),
+    url(r"^translations/", views.translations, name="pontoon.translations"),
```

Comment:
I'd make that less ambiguous. Maybe `views.download_translations` and `pontoon.download.translations`?

Reply:
* Yeah , i should have taken a hint from ```download_translation_memory```